src/AbstractInRangeParser.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+abstract class AbstractInRangeParser extends AbstractValueParser
+{
+    protected $minValue;
+    protected $maxValue;
+
+    public function __construct(Config $config, $name, $value, $minValue, $maxValue)
+    {
+        $this->minValue = $minValue;
+        $this->maxValue = $maxValue;
+        parent::__construct($config, $name, $value);
+    }
+
+    protected function parse($value)
+    {
+        if (!is_numeric($value)) {
+            return null;
+        }
+
+        if ($this instanceof IntInRangeParser) {
```

Comment:
@mpscholten, the only thing I'm not 100% sure about, is the implementation of `parse()` in the abstract class.
On the one hand, it's convenient to write it once here and then just:
```php
return parent::parse($value);
```
On the other hand, it sort of makes `AbstractInRangeParser` opinionated with respect to its actual instance... what do you think?